pineappl_py/tests/test_grid.py

```diff
@@ -212,6 +212,55 @@ def test_bins(self, fake_grids):
         np.testing.assert_allclose(g.bin_left(1), [2, 3])
         np.testing.assert_allclose(g.bin_right(1), [3, 5])
 
+    def test_rotate_pidbasis(self, fake_grids):
+        g = fake_grids.grid_with_generic_convolution(
+            nb_convolutions=2,
+            channels=CHANNELS,
+            orders=ORDERS,
+            convolutions=[CONVOBJECT, CONVOBJECT],
+        )
+        # Rotate the Grid into the PDG basis
+        g.rotate_pid_basis(PidBasis.Pdg)
+        assert g.pid_basis == PidBasis.Pdg
```

Comment:
you need to check the convolution numbers stay the same

Reply:
Addressed in [adeb134](https://github.com/NNPDF/pineappl/pull/299/commits/adeb134960eb112e6d72e32e1ae6c1721a62f435).